src/KineticsStripOutputNode.cpp

```diff
@@ -0,0 +1,102 @@
+#include "KineticsStripOutputNode.h"
+#include <iostream>
+#include <algorithm>
+
+// Reusable state for sending data packets
+static int out_fd = -1;
+// static struct sockaddr_in out_addr;
+
+// This is dynamic because we need to allocate memory based on
+// the number of pixels per strip.
+unsigned char out_packet[512] = {};
+
+std::string ip;
+int startport, width, height;
+struct sockaddr_in out_addr;
```

Comment:
Can you also change these names to be camelCase

Reply:
whoops, can you tell I ported the code from python?